lib/node/failure_detector.ml

```diff
@@ -168,3 +170,37 @@ let failure_detection node =
           Lwt_unix.sleep @@ Float.of_int t.config.protocol_period;
         ] in
     Lwt.return ()
+
+module PeerSet = Set.Make (struct
+  let compare = Stdlib.compare
```

Comment:
How exactly does using `Stdlib.compare` for `Peer.t` work?

Reply:
Good question!
I guess adding some tests would help us to determine this :)

Reply:
Outdated, this `set` does not exist anymore :)